### ISSUE TITLE
getUserMedia always prompts for audio recording and image/video capture permissions regardless of constraints

### ISSUE THREAD

**Issue by markisadesignerd** [phonegap/phonegap-plugin-media-stream#8]
### Expected Behaviour
If I call `getUserMedia` and pass in a `MediaStreamConstraints` that only requests `audio`, I should not be prompted for image/video capture permissions. 

### Actual Behavior
Regardless of constraints, audio and image/video capture permissions are always requested.

#### Steps to Reproduce
Call `getUserMedia` with a `MediaStreamConstraints` object where `video` is `false`.
e.g.
```js
{
  audio: true,
  video: false
}
```

#### Platform and Version (eg. Android 5.0 or iOS 9.2.1)
Android 8.0
Android 6.0.1

#### (Android) What device vendor (e.g. Samsung, HTC, Sony...)
Google Pixel
LG Nexus 5

#### Cordova CLI version and cordova platform version
`cordova --version`
7.1.0

`cordova platform version android `
6.3.0

#### Plugin version
phonegap-plugin-media-recorder 1.1.0 "Media Recorder"
phonegap-plugin-media-stream 1.2.0 "MediaStream"

#### Sample Code that illustrates the problem
```js
navigator.mediaDevices.getUserMedia({
  audio: true,
  video: false
})
// Both audio recording and image/video capture permissions are requested
  .then(mediastream => {
    // record audio
  })
```
#### Logs taken while reproducing problem
n/a

**Comment by macdonst** [comment]
@markisadesignerd you are correct. We'll get that fixed soon.

**Comment by macdonst** [comment]
@markisadesignerd can you try installing from `master`? I believe I've fixed the issue. Thanks for reporting the bug.

**Comment by markisadesignerd** [comment]
Works great, thanks!

**Comment by macdonst** [comment]
@markisadesignerd great, I'm going to close this but I should release a new point release this week.

**Comment by macdonst** [comment]
@markisadesignerd released a 1.2.1
